backend/controller/ingress/ingress.go

```diff
@@ -249,3 +254,46 @@ func validateValue(fieldType schema.Type, path path, value any, sch *schema.Sche
 	}
 	return nil
 }
+
+func parseQueryParams(values url.Values) (map[string]any, error) {
+	if jsonStr, ok := values["json"]; ok {
```

Comment:
I think this should check that `json` is the only parameter too.

Reply:
I think we should name this something that can't also be a field name, just in case a user has a field named `json`... probably an edge case TBH, but if we can avoid the ambiguity altogether it would be good. Maybe `@json`?